ci/scripts/cpp_test.ps1

```diff
@@ -18,16 +18,8 @@
 
 $ErrorActionPreference = "Stop"
 
-$SourceDir = $Args[0]
-$BuildDir = $Args[1]
-$InstallDir = if ($Args[2] -ne $null) { $Args[2] } else { Join-Path $BuildDir "local/" }
-
-$BuildAll = $env:BUILD_ALL -ne "0"
-$BuildDriverManager = ($BuildAll -and (-not ($env:BUILD_DRIVER_MANAGER -eq "0"))) -or ($env:BUILD_DRIVER_MANAGER -eq "1")
-$BuildDriverFlightSql = ($BuildAll -and (-not ($env:BUILD_DRIVER_FLIGHTSQL -eq "0"))) -or ($env:BUILD_DRIVER_FLIGHTSQL -eq "1")
-$BuildDriverPostgreSQL = ($BuildAll -and (-not ($env:BUILD_DRIVER_POSTGRESQL -eq "0"))) -or ($env:BUILD_DRIVER_POSTGRESQL -eq "1")
-$BuildDriverSqlite = ($BuildAll -and (-not ($env:BUILD_DRIVER_SQLITE -eq "0"))) -or ($env:BUILD_DRIVER_SQLITE -eq "1")
-$BuildDriverSnowflake = ($BuildAll -and (-not ($env:BUILD_DRIVER_SNOWFLAKE -eq "0"))) -or ($env:BUILD_DRIVER_SNOWFLAKE -eq "1")
```

Comment:
Ah, we do actually want this though, because things like the verification scripts use this so we can build all the projects but only test some of them (since we can't reasonably expect people to have Snowflake credentials, for example)

Reply:
Will take another look. Shouldn't these be in the build script not the test script if anything though?

Reply:
We want to build all of the drivers, but only run some of the tests.

I suppose we could make it so that we don't build the tests in the first place, but for verification it'd be nice to at least validate that everything builds.

Reply:
Ah I see. I missed that these were being used for ctest labels - just assumed we ran a blanket `ctest` and whatever was built prior to the test execution is what gets tested. Makes sense on second review